pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -15,6 +17,17 @@ static __always_inline int is_ephemeral_port(u16 port) {
     return port >= EPHEMERAL_RANGE_BEG && port <= EPHEMERAL_RANGE_END;
 }
 
+static __always_inline void read_skb_data(struct __sk_buff* skb, u32 offset, char *buffer) {
+    if (skb->len - offset < HTTP_BUFFER_SIZE) {
```

Comment:
Do we need to handle the case where `skb->len < HTTP_BUFFER_SIZE`?

Reply:
good catch. Ideally yes, but I recall struggling to make it work with the Kernel 4.4 verifier. I think for now it's fine to skip TCP segments that are smaller than `HTTP_BUFFER_SIZE` because I can't think of a realistic scenario where you'll have a request or response smaller than that, but we should revisit this when we have runtime compilation.